### PR TITLE
Funding 2021: Created Astroquery Maintenance proposal

### PR DESCRIPTION
**Pull request by keflavich**
This is a proposal to support maintenance of astroquery.

It's current a work in progress - I don't have final $ requests yet.  I'm submitting the WIP to make sure this is really what was intended in the [instructions](https://github.com/astropy/astropy-project/blob/master/finance/proposal-calls/Moore_2021.md) sent by @hamogu.

### CODE REVIEW

**Comment by kelle** [comment]
This looks very much on track to me -- thanks for getting things started!

Regarding the actual idea, I think it would be useful to know if this funding would be to support about the same amount of time that y'all are already dedicating or if this funding would enable some of you to dedicate more time. Either way, I think it will be a very compelling, I would just like the expectations to be as clear as possible.

**Comment by keflavich** [comment]
Alright, we can add that.  For me, it's same amt of time.  For Brigitta and Clara, we're still working that out - but it's not clear we'll know the payment path (and therefore the funding requirements and available time) by March 20.  If we put in a ballpark estimate, can we iterate on the total commitment with the finance committee later?  Is it better to over- or under-ask, for your planning purposes? (of course, an exact number is best, but that may not be available)

Is this the right folder / location in this repo to put the document?

**Comment by kelle** [comment]
I think it would be best to give the range. E.g., bracket the budget and let folks know the range of the potential cost. At this point, I think it's most important that the **community** have a good sense of the order of magnitude of what is being requested. The implementation details and exact costs can be worked out with the finance committee later.

I don't currently have any opinions about the location of these PRs.

**Comment by jdswinbank** [comment]
> Is this the right folder / location in this repo to put the document?

Yes, this is a good location. 👍

File: finance/proposal-calls/2021-proposal/AstroqueryMaintenance.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### Budget
+currency: US $
+Include overheads, fringe etc. if money is paid as sub-grant (note that the Moore grant caps overhead at 15%).
+
+- Salary:
+
+ * Adam Ginsburg: 1/12th FTE (1 month effort), $19000 including overheads (though this may need to be renegotiated re: Moore limitations)
+ * Brigitta Sipocz: $???
+ * Clara Brasseur: $???
```
Comment (ceb8) [inline finance/proposal-calls/2021-proposal/AstroqueryMaintenance.md:25]:
```suggestion
 * Clara Brasseur: 1/12th FTE (1 month effort), $15000 including overheads
```

**Comment by ceb8** [comment]
@kelle  For me this would be the same amount of time, just next year I won't be supported by STScI for the work as I am now (starting a PhD program).

**Comment by bsipocz** [comment]
My part from this maintenance proposal is included in #192 along with new projects for astroquery. This is in agreement within the maintainer group for easier logistic, the two PRs are not in competition with each other and could be merged if preferred.

File: finance/proposal-calls/2021-proposal/AstroqueryMaintenance.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Communicate with external development teams
+
+### Budget
+currency: US $
+Include overheads, fringe etc. if money is paid as sub-grant (note that the Moore grant caps overhead at 15%).
+
+- Salary:
+
+ * Adam Ginsburg: 1/12th FTE (1 month effort), $19000 including overheads (though this may need to be renegotiated re: Moore limitations)
+ * Brigitta Sipocz: $???
```
[the remark this answers is not in the document]
Comment (bsipocz) [reply finance/proposal-calls/2021-proposal/AstroqueryMaintenance.md:24]: Yes, sounds good. Thanks

**Comment by jdswinbank** [comment]
Please react to this comment to vote on this proposal (:+1:, :-1:, or no reaction for +0).

**Comment by eteq** [comment]
Congratulations @keflavich!  This proposal has the support of the Astropy Community and has been awarded  $27,200 to implement.  This PR will be merged, and you can contact finance@astropy.org or reach out on Slack to arrange financial details (see more details in the email to astropy-dev).

**Pull request merged**